lib/matplotlib/backends/backend_agg.py

```diff
@@ -272,7 +272,9 @@ def _prepare_font(self, font_prop):
         """
         Get the `.FT2Font` for *font_prop*, clear its buffer, and set its size.
         """
-        font = get_font(findfont(font_prop))
+        fname = find_fonts_by_props(font_prop)
```

Comment:
`fname` should be renamed to `filename_dict` or at least `filenames` (plural).

Reply:
I do not remember why we used a dictionary instead of a list